spec/debug_logging/instance_notifier_spec.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 require 'spec_helper'
+require 'debug_logging/active_support_notifications'
```

Comment:
We should remove this.  So long as the setter is working as intended this is no longer needed.
```suggestion
```

Reply:
Yup, I replaced the `require 'debug_logging/active_support_notifications'` method with calls to the setter (`DebugLogging.configuration.active_support_notifications = true`) in each of the three AS::N related tests:

`spec/debug_logging/class_notifier_spec.rb`
`spec/debug_logging/instance_notifier_spec.rb`
`spec/debug_logging/log_subscriber_spec.rb`